.github/FUNDING.yml

```
@@ -0,0 +1,12 @@
+# These are supported funding model platforms
+
+github: fluentpos
+patreon: # Replace with a single Patreon username
+open_collective: fluentpos
```

```suggestion
github: # Replace with a GitHub username
patreon: # Replace with a single Patreon username
open_collective: # Replace with a open collective username
```

This file can likely be removed.